playbooks/portals-setup-following.yml

```diff
@@ -13,11 +13,8 @@
     - name: Check '--limit' is used
       include_tasks: tasks/host-limit-check.yml
 
-    - name: Check supported OS and version
-      assert:
-        that:
-          - ansible_distribution|lower == 'debian' and ansible_distribution_version == '10'
-        fail_msg: "{{ ansible_distribution }} {{ ansible_distribution_version }} is not yet supported by this role"
+    - name: Check server OS compatibility
```

Comment:
same

Reply:
renamed in https://github.com/SkynetLabs/ansible-playbooks/pull/238/commits/de6d344272248acf2313779c0dbc53758035699c